### DISCUSSION TITLE
UUP Converter

### DISCUSSION THREAD

**Discussion by gassilin** [utmapp/UTM#5394] [Windows]
Exporting Windows 11 Core to install.wim...
[ERROR] Destination image description contains illegal characters
ERROR: Exiting with error code 24:
       An invalid parameter was given.
Failed to export Windows 11 Core to install.wim

**Configuration**  
* UTM Version: 4.2.5 (81)
* macOS Version: ventura 13.4
* Mac Chip (Intel, M1, ...):  m1

**Comment by johann-lau** [comment]
Same issue here. Are the developers going to do something or just ignore it after moving it to a discussion?

**Comment by GeorgeShiZZZ** [comment]
I have the same issue. Anyone got it fixed?

**Reply by GeorgeShiZZZ** [reply]
Try Windows with English instead of Chinese. I can't export with Chinese, either. However, I can with English.

**Comment by ebiggers** [comment]
This is fixed in wimlib v1.14.3-BETA1.  See https://wimlib.net/forums/viewtopic.php?p=1609

**Reply by zhucaidan** [reply]
I still having this issue, how should I integrate bew wimlib into UUP Converter?

**Reply by osy** [reply]
Use CrystalFetch instead https://github.com/TuringSoftware/CrystalFetch/tree/main

**Reply by ebiggers** [reply]
wimlib v1.14.3 was released with the fix.
